new-dti-website-redesign/src/components/TeamCard.tsx

```diff
@@ -0,0 +1,136 @@
+import { getColorClass, productLinks } from '@/utils/memberUtils';
+import Image from 'next/image';
+import Link from 'next/link';
+
+const MemberSummary = ({
+  user,
+  image,
+  enlarged = false
+}: {
+  user: IdolMember;
+  image: string;
+  enlarged?: boolean;
+}) => (
+  <>
+    <Image
+      src={image}
+      alt={`${user.firstName} ${user.lastName}`}
+      className="rounded-lg w-auto h-auto"
+      width={232}
+      height={232}
+    />
+    <div className="text-left">
+      {enlarged ? (
+        <h3>{`${user.firstName} ${user.lastName}`}</h3>
+      ) : (
+        <h6>{`${user.firstName} ${user.lastName}`}</h6>
+      )}
+      <p className={`text-${getColorClass(user.role)}`}>{user.roleDescription}</p>
+    </div>
+  </>
+);
+
+type MemberCardProps = {
+  user: IdolMember;
+  image: string;
+  selected: boolean;
+  onClick?: () => void;
+};
+
+export const MemberCard = ({ user, image, selected, onClick }: MemberCardProps) => {
+  const baseStyles =
+    'p-8 border border-border-1 flex flex-col gap-4 focusState hover:shadow-white hover:shadow-[0_-4px_8px_0]';
```

Comment:
Ah sorry if this wasn't clear enough in the [specs](https://www.figma.com/design/ttAGEX3pHmzuhMzp8uAyhz/SP25-Cornelldti.org-Website-Revamp?node-id=1336-13031&m=dev) – I was thinking that on hover the card has a background color change (the shadow only appears when the card is selected/open)


```suggestion
    'p-8 border border-border-1 flex flex-col gap-4 focusState hover:bg-background-2 transition-[background-color] duration-[120ms]';
```

also added a transition that is consistent with the button hover states